src/server/twitterSearch.js

```diff
@@ -23,43 +33,82 @@ const makeTweeterFromRaw = (raw) => {
     .build();
 };
 
-export const searchAndSave = (res, query) => {
-  T.get('search/tweets', { 'q': query, 'count': 300 }, function (err, result, response) {
-    let count = 0;
-    result.statuses.forEach((tweetRaw) => {
-      const userRaw = tweetRaw.user;
-
-      const tweet = Builders.TweetBuilder()
-        .id(tweetRaw.id)
-        .content(tweetRaw.text)
-        .date(moment(new Date(tweetRaw.created_at)).format('YYYY-MM-DD HH:mm:ss'))
-        .likes(tweetRaw.favourite_count || 0)
-        .retweets(tweetRaw.retweet_count || 0)
-        .build();
+const processTweet = (tweetRaw, isRetweet = false) => {
+  const tweeter = makeTweeterFromRaw(tweetRaw.user);
+  const retweetedStatusRaw = tweetRaw.retweeted_status;

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'retweeted_status' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2175579853/issues/source?bid=3148782&fileBranchId=3163167#l38)